IndentationError: unindent does not match any outer indentation level
False: Non-zero return code, probably a syntax error occurred\ntime=1607548428.0064054 gym-duckietown 2019.0.0

time=1607548428.363328 Registering gym environment id: Duckietown-4way-v0
time=1607548428.3635488 Registering gym environment id: Duckietown-loop_dyn_duckiebots-v0
time=1607548428.3636422 Registering gym environment id: Duckietown-loop_empty-v0
time=1607548428.363721 Registering gym environment id: Duckietown-loop_obstacles-v0
time=1607548428.3637955 Registering gym environment id: Duckietown-loop_pedestrians-v0
time=1607548428.3638725 Registering gym environment id: Duckietown-small_loop-v0
time=1607548428.3639479 Registering gym environment id: Duckietown-small_loop_cw-v0
time=1607548428.364029 Registering gym environment id: Duckietown-straight_road-v0
time=1607548428.3641067 Registering gym environment id: Duckietown-straight_road_10_tiles_no_start_tile-v0
time=1607548428.3643112 Registering gym environment id: Duckietown-straight_road_no_start_tile-v0
time=1607548428.3644001 Registering gym environment id: Duckietown-straight_road_trafficlight-v0
time=1607548428.3644876 Registering gym environment id: Duckietown-straight_road_with_side_floor-v0
time=1607548428.3645713 Registering gym environment id: Duckietown-udem1-v0
time=1607548428.36465 Registering gym environment id: Duckietown-zigzag_dists-v0
Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/appuser/aido-auto/gym-duckietown/gym_duckietown/tasks/lf_challenge_no_cv/checker.py", line 6, in <module>
    checkers=[RewardChecker()]
  File "/home/appuser/aido-auto/gym-duckietown/gym_duckietown/tasks/solution_evaluator.py", line 31, in __init__
    solution_file_as_module = importlib.import_module(self.args.solution_file_as_module)
  File "/usr/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 674, in exec_module
  File "<frozen importlib._bootstrap_external>", line 781, in get_code
  File "<frozen importlib._bootstrap_external>", line 741, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/appuser/aido-auto/gym-duckietown/gym_duckietown/tasks/lf_challenge_no_cv/solutions/solution.py", line 11
    def control(self, delta):
                            ^
IndentationError: unindent does not match any outer indentation level
